Source/GTMSessionFetcher.m

```diff
@@ -92,10 +92,21 @@
   #endif
 #endif
 
-#if ((TARGET_OS_IPHONE && defined(__IPHONE_13_0) && __IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_13_0) || \
-     (TARGET_OS_OSX && defined(MAC_OS_X_VERSION_10_15) && MAC_OS_X_VERSION_MAX_ALLOWED >= MAC_OS_X_VERSION_10_15))
+#if ((defined(TARGET_OS_MACCATALYST) && TARGET_OS_MACCATALYST) || \
+     (TARGET_OS_OSX && defined(__MAC_10_15) && __MAC_OS_X_VERSION_MIN_REQUIRED >= __MAC_10_15) || \
+     (TARGET_OS_IOS && defined(__IPHONE_13_0) && __IPHONE_OS_VERSION_MIN_REQUIRED >= __IPHONE_13_0) || \
```

Comment:
are there `IOS` constants that should be use for all these rather than the mix of `IPHONE` and `IOS` ones?  (just thinking about how watchOS sdks still defines the `MIN_REQUIRED` one, but pinned to something old.

Reply:
I don't see ones besides the __IPHONE ones.

Regarding the watchOS SDK defining the iPhone min version, that only happens when the watch is targeted (they must need the iPhone min defined for that), so because we first check the targeted OS it should be fine.